src/_variables.scss

```diff
@@ -1,12 +1,14 @@
-$lightest-blue: #e7f0ff;
-$lighter-blue: #d3e3fc;
-$light-blue: #b1cefd;
-$blue: #77a6f7;
-$teal: #00887a;
-$baby-blue: #00bad3;
-$light-red: #fd8383;
-$lighter-red: #ffccbc;
-$shadow-grey: #777;
+// $lightest-blue: #e7f0ff;
```

Comment:
Why are these commented out?

Reply:
I did this for debugging purposes, but it will be reverted.